app/models/user.rb

```diff
@@ -1,5 +1,5 @@
-class User < ActiveRecord::Base
-  validates_presence_of :provider, :uid, :name, :oauth_token, :oauth_expires_at
+class User < ApplicationRecord
+  validates :provider, :uid, :name, :oauth_token, :oauth_expires_at, presence: true
 
```

Comment:
idk why i made this change, just trying things out

Reply:
Both accomplish the same thing. It's up to your style really.